### PR TITLE
Colab Notebook for Normalizations

### PR DESCRIPTION
**Pull request by Smokrow**
First draft of a Colab Notebook for Normalization layers. 

I tried to formulate the description very basic and non technical for non ML experts while also including some more advanced topics like initialisers for example. 

I will probably add a Recurrent example for one of the layers (Maybe layer normalization) as well.

### CODE REVIEW

**Comment by armando-fandango** [comment]
Thanks @Smokrow Looks great. @seanpmorgan @facaiy should the ipynb files be in examples folder ? Because /docs folder would be overwritten everytime we run API docs generator.

**Comment by Smokrow** [comment]
Good point.  I will move it

File: tensorflow_addons/examples/Normalizations.ipynb [hunk 1]
```diff
@@ -0,0 +1,126 @@
+{
```
Comment (seanpmorgan) [inline tensorflow_addons/examples/Normalizations.ipynb]: <p>Please move this to `addons/examples/jupyter_notebooks` <a href="https://github.com/tensorflow/probability/tree/master/tensorflow_probability/examples/jupyter_notebooks" target="_blank">as is done in tensorflow probability</a></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (seanpmorgan) [inline tensorflow_addons/examples/Normalizations.ipynb]: <p>Can you try to match the `tensorflow/docs` template as much as possible:</p><p><a href="https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb" target="_blank">https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb</a></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (seanpmorgan) [inline tensorflow_addons/examples/Normalizations.ipynb]: <p>It looks like this PR has two notebooks. Could you please remove one and move the more complete example to the directory mentioned above</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by Smokrow** [comment]
@ppwwyyxx as promised the colab notebook we talked about #14 . Would you mind taking a look?

Do you know a good example we can add to showcase the axis parameter? (I tried using channels first/channels last but this will not run on colab since we have no GPU on there)

**Comment by seanpmorgan** [comment]
Hmmm.. Colab should have a GPU, but our pip package has not implemented GPU kernels yet (See #118). This is also a good time to make sure that axis parameter is intuitive because @ppwwyyxx expressed concerns about it not really making sense.

**Comment by seanpmorgan** [comment]
@Smokrow when time allows:
1. Remove the duplicate notebook
2. Move the notebook to just `tensorflow_addons/examples`
3. Rename the notebook to `layers_normalizations`
4. Edit the colab link to be the location the notebook will be at once it's merged.
5. Update the content now the LayerNorm has been removed
6. Provide an end to end example of GroupNorm vs. BatchNorm or LayerNorm from core

**Comment by Smokrow** [comment]
We mark this PR as WIP until tf core adds a normalization layer. After that I will change the links in the notebook since the comparison is not working right now

**Comment by seanpmorgan** [comment]
@Smokrow Just FYI this should be unblocked now that we are built against tf2-beta

**Comment by Smokrow** [comment]
Everything seems to be working fine now 
Feel free to merge. 

Sry for the long wait
